draft-ietf-quic-recovery.md

```diff
@@ -302,11 +302,10 @@ Retransmission Timeout mechanisms.
 ### Crypto Handshake Timeout
 
 Data in CRYPTO frames is critical to QUIC transport and crypto negotiation, so a
-more aggressive timeout is used to retransmit it.  Below, the term "handshake
-packet" is used to refer to packets containing CRYPTO frames, not packets with
-the specific long header packet type Handshake.
+more aggressive timeout is used to retransmit it.  Below, the term "crypto
+packet" is used to refer to packets containing CRYPTO frames.
```

Comment:
We should further qualify this: "Handshake or Initial packets containing CRYPTO frames", because CRYPTO frames after the handshake completes don't require special handling.

Reply:
+1

Reply:
Done, but is the term "handshake complete" well-defined?  I wanted to use long header packets, but I realized that the CFIN is head of line blocking :(